backend_app/github_retrieval/requirements.txt

```diff
@@ -0,0 +1,45 @@
+autopep8==1.5
+bcrypt==3.1.7
+certifi==2019.11.28
+cffi==1.13.2
+chardet==3.0.4
+Click==7.0
+cycler==0.10.0
+Flask==1.1.1
+Flask-Cors==3.0.8
+future==0.18.2
+googlemaps==4.0.0
+httplib2==0.17.0
+idna==2.8
+itsdangerous==1.1.0
+Jinja2==2.11.1
+joblib==0.14.1
+kiwisolver==1.1.0
+MarkupSafe==1.1.1
+matplotlib==3.1.2
+numpy==1.18.1
+oauth2client==4.1.3
+pandas==0.25.3
+parsedatetime==2.5
+pbr==5.4.4
+pyasn1==0.4.8
+pyasn1-modules==0.2.8
+pycodestyle==2.5.0
+pycparser==2.19
+pyparsing==2.4.6
+python-dateutil==2.8.1
+pytz==2019.3
+requests==2.22.0
+rsa==4.0
+scikit-learn==0.22.1
+scipy==1.4.1
+six==1.14.0
+smartcar==4.2.4
+stevedore==1.31.0
+tqdm==4.42.1
```

Comment:
This is the only import that should be in the Requirements.txt file. The rest of these can be deleted.

Reply:
Yes. I agree with you. I will go ahead and update the pull request

Reply:
I have updated it